StabilityPackage.wl

```diff
@@ -372,6 +372,27 @@ Export[ToString[name]<>".h5",  {
 "/settings/inputs"-> {"Data"-> ToString[outevs[[3]]],"Attributes"-> {"Order"-> "file,rsrt,rend,testE,hi,nstep"}}
 }
 ];
+(*

```

Comment:
Good job with the documentation. Even better would be separate functions for separate outputs, but we'll tackle that another day.

Reply:
Yeah, I didn't know if there was a standard way to document list dimensions/entries so I tried to be as explicit as possible.